### PR TITLE
Create water-css

### PR DESCRIPTION
**Pull request by kylejrp**
This is a farily minimal CSS framework with some good opportunities to contribute!

### CODE REVIEW

**Comment by ritwik12** [comment]
@kylejrp Your filename should be "water-css.yml"

File: _data/projects/water-css [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+site: https://watercss.netlify.com/
+tags:
+- css
+- sass
+- scss
+- javascript
+- html
+upforgrabs:
+  name: help-wanted
+  link: https://github.com/kognise/water.css/labels/good%20first%20issue
```
Comment (ritwik12) [inline _data/projects/water-css]: @kylejrp You gave label name "help wanted" above and then gave the link for "good first issue", Please use both name and link for one of them only.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/water-css (hunk 1 above)
Comment (kylejrp) [reply _data/projects/water-css]: Thanks, I updated the link to be for the help-wanted label.

**Comment by kylejrp** [comment]
I've made your requested changes - let me know if there's anything else I need to do.

Thanks for your help and patience!

**Comment by ritwik12** [comment]
@kylejrp Thanks!!

**Pull request merged**